subsys/logging/Kconfig

```diff
@@ -4,6 +4,7 @@
 menuconfig LOG
 	bool "Logging"
 	select PRINTK if USERSPACE
+	select UART_CONSOLE if X86_64
```

Comment:
There may be some use cases where it makes sense to not use this option.

Reply:
Good point, I am still trying narrow this down.

Reply:
Got this fixed, thanks.